### ISSUE TITLE
[ng] address broken links in angular docs

### ISSUE THREAD

**Issue by chalin** [dart-lang/site-webdev#204]
Subsumes #202

**Comment by chalin** [comment]
Reopening. There are still broken links _into_ the API docs. Under angular.io, this was handled via a simple URL rewrite rule. Maybe something similar can be used here.

**Comment by chalin** [comment]
Some examples of remaining broken links.

Links within elements having `if-docs="ts"` directives:
```
https://webdev.dartlang.org/angular/guide
- (556:21) 'Change Log' => https://webdev.dartlang.org/angular/guide/change-log.html (HTTP 301 => 404)
  - redirect path:
    - https://webdev.dartlang.org/angular/guide/change-log.html (301)
    - /angular/guide/change-log (404)
```

There are link errors for `guide/router` (which is still being written) because of missing anchors:
```
https://webdev.dartlang.org/angular/tutorial/toh-pt5
- (738:84) 'Router' => https://webdev.dartlang.org/angular/guide/router.html#base-href (HTTP 200 but missing anchor)
  - redirect path:
    - https://webdev.dartlang.org/angular/guide/router.html (301)
    - /angular/guide/router (200)
```

**Comment by chalin** [comment]
Non-API issues reported by the link checker were addressed in #249. The only remaining issues being reported by the link checker are with API pages, which is covered by #216. So I'll (re-)close this issue.
